library.sh

```diff
@@ -794,11 +794,16 @@ function shellcheck_new_files() {
 }
 
 function latest_version() {
-  local semver=$(git describe --match "v[0-9]*" --abbrev=0)
-  local major_minor=$(echo "$semver" | cut -d. -f1-2)
+  if [ $(current_branch) = "master" ]; then
```

Comment:
My git fu is not strong enough to reason about the actual PR, sorry... But the one thing we might want to do is include:
branch = "main"
here since we're moving away from using "master" so that would make this future proof.

Reply:
Make sense. Fixed as suggested.